packages/mobile/src/screens/edit-track-screen/screens/CoverAttributionScreen.tsx

```diff
@@ -0,0 +1,73 @@
+import { useField } from 'formik'
+
+import { Box, Divider, Flex } from '@audius/harmony-native'
+import { Switch, Text } from 'app/components/core'
+import { HarmonyTextField } from 'app/components/fields'
+import { FormScreen } from 'app/screens/form-screen'
+
+const messages = {
+  title: 'Cover Attribution',
+  isCover: 'This Song is a Cover',
+  description: 'This track was written by another artist.',
+  attribution: {
+    header: 'Please provide the original song title and artist name.',
+    originalSongTitle: 'Original Song Title',
+    originalArtist: 'Original Artist'
+  }
+}
+
+export const COVER_ORIGINAL_SONG_TITLE = 'cover_original_song_title'
+export const COVER_ORIGINAL_ARTIST = 'cover_original_artist'
+
+export const CoverAttributionScreen = () => {
+  const [{ value: isCover }, _ignored, { setValue: setIsCover }] =
+    useField<boolean>('isCover')
+  const [{ value: originalSongTitle }] = useField(COVER_ORIGINAL_SONG_TITLE)
+  const [{ value: originalArtist }] = useField(COVER_ORIGINAL_ARTIST)
+
+  return (
+    <FormScreen title={messages.title} variant='white'>
+      <Box ph='xl' mt='xl'>
+        <Box>
+          <Flex
+            direction='row'
+            justifyContent='space-between'
+            alignItems='center'
+            mb='s'
+          >
+            <Text fontSize='large' weight='demiBold'>
+              {messages.isCover}
+            </Text>
+            <Switch value={isCover} onValueChange={setIsCover} />
+          </Flex>
+          <Text variant='body'>{messages.description}</Text>
+        </Box>
+
+        {isCover ? (
+          <Box>
+            <Divider mt='xl' />
```

Comment:
instead of all the `mt`'s is there a way we can use the `gap` prop instead?

Reply:
gap doesn't work in this instance, we'll still need something to move the divider away from the top of the box. the gap will just seperate the two children elements

Reply:
fair nuf. could probly do something like below, but splitting hairs at this point:
<Box pv='xl gap='xl>
<Divider/>
<Box>
stuff